cat-launcher/src/pages/BackupsPage/DeleteBackupDialog.tsx

```diff
@@ -0,0 +1,40 @@
+import {
```

Comment:
Create a common and generic Dialog component and move it to the components directory. Use that component in DeleteBackupDialog as well as the RestoreBackupDialog to reduce boilerplate and repetitive code.